src/MinterGateway.sol

```diff
@@ -410,16 +410,12 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
     function deactivateMinter(address minter_) external onlyActiveMinter(minter_) returns (uint240 inactiveOwedM_) {
         if (isMinterApproved(minter_)) revert StillApprovedMinter();
 
-        uint112 principalOfActiveOwedM_ = principalOfActiveOwedMOf(minter_);
+        _imposePenaltyIfMissedCollateralUpdates(minter_);
```

Comment:
too many changes. we can add penalty principal into `DeactivateMinter` event. It should be sufficient.

Reply:
How is this too many changes? It's 1 line and then renaming `principalOfActiveOwedM_` to `principalOfActiveOwedM_`.

Reply:
If we don't trust `_imposePenaltyIfMissedCollateralUpdates` here, then we don't trust it anywhere.

Reply:
fair, the code looks good. Just moving pieces after everyone re-reviewed X times make me worried

Reply:
let's change this thing, but it 100% last thing that is changed =)